connect/models/asset.py

```diff
@@ -3,14 +3,14 @@
 # This file is part of the Ingram Micro Cloud Blue Connect SDK.
 # Copyright (c) 2019 Ingram Micro. All Rights Reserved.
 
-from marshmallow import fields, post_load
 from typing import List, Optional
 
-from .base import BaseModel, BaseSchema
-from .connection import Connection, ConnectionSchema
-from .parameters import Param, ParamSchema
-from .product import Item, ItemSchema, Product, ProductSchema
-from .tiers import Tiers, TiersSchema
+from .base import BaseModel
+from .connection import Connection
+from .parameters import Param
+from .product import Item, Product
+from .tiers import Tiers
```

Comment:
In general tiers is agains singular for all other parts 
TierConfig 
TierAccount 
...
There we have like Array of TierAccounts but indexed by customer/tier1/tier2 ...
so probably is ok but probably better TierAccounts?

Reply:
Renamed Tier -> TierAccount and Tiers -> TierAccounts. Removed Account model since it was the same as TierAccount.